platform_design/lib/main.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+
+import 'tab_1.dart';
+import 'tab_2.dart';
+import 'tab_3.dart';
+import 'tab_4.dart';
+import 'widgets.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  final platformOverride = ValueNotifier<TargetPlatform>(defaultTargetPlatform);
+
+  @override
+  Widget build(BuildContext context) {
+    // Change this value to better see animations.
+    timeDilation = 1;
+    // Either Material or Cupertino widgets work in either Material or Cupertino
+    // Apps.
+    return ValueListenableBuilder(
+      valueListenable: platformOverride,
+      child: PlatformAdaptingHomePage(platformOverride: platformOverride),
+      builder: (BuildContext context, TargetPlatform platform, Widget child) {
+        return MaterialApp(
+          title: 'Adaptive Music App',
+          theme: ThemeData(
+            // Use the green theme for Material widgets.
+            primarySwatch: Colors.green,
+            platform: platform,
+          ),
+          builder: (BuildContext context, Widget child) {
+            return CupertinoTheme(
+              // Instead of letting Cupertino widgets auto-adapt to the Material
+              // theme (which is green), we're going to use a different theme
```

Comment:
Do you mean use a passive tone instead?

Reply:
I usually make the app or the code itself the star:

"Rather than letting Cupertino widgets auto-adapt to the Material theme (which is green), the app uses a different theme for Cupertino (which is blue by default)."

Reply:
Ohh I get it. Nice, I like it

(I'm just trying to sneak a few of these in per day)

Reply:
removed all the wes elsewhere in the code